ravenframework/SupervisedLearning/ScikitLearn/SVM/SVR.py

```diff
@@ -113,6 +115,9 @@ def _handleInput(self, paramInput):
     settings, notFound = paramInput.findNodesAndExtractValues(['C', 'kernel', 'degree', 'gamma', 'coef0',
                                                              'tol', 'cache_size', 'epsilon', 'shrinking', 'max_iter',
                                                              'verbose'])
+
+    setting,_ = paramInput.findNodesAndExtractValues(['normalizeTargets'])
+    self.info['normalizeTargets'] = setting['normalizeTargets']
```

Comment:
I think we can create a method to check if we want to perform normalization or not. For SVR, we can compute the ratio (basically, the normalized parameters and the default parameters), if the normalized parameters are too large, we can provide normalization on the targets.